dotCMS/src/main/webapp/html/portlet/ext/browser/view_browser_js_inc.jsp

```diff
@@ -1328,17 +1328,15 @@ Structure defaultFileAssetStructure = CacheLocator.getContentTypeCache().getStru
     }
 
     function moveFolder (objId, parentId, referer) {
-        //top.location='<portlet:actionURL windowState="<%= WindowState.MAXIMIZED.toString() %>"><portlet:param name="struts_action" value="/ext/folders/edit_folder" /><portlet:param name="cmd" value="move" /></portlet:actionURL>&inode=' + objId + '&parent=' + parentId + '&referer=' + referer;
         BrowserAjax.moveFolder(objId, parentId, moveFolderCallback);
-        setTimeout('reloadContent()',500);
     }
 
     function moveFolderCallback (response) {
+        reloadContent();
         if(response == '<%= UtilMethods.escapeSingleQuotes(LanguageUtil.get(pageContext, "Folder-moved")) %>'){
             BrowserAjax.getTree(null, initializeTree);
             showDotCMSSystemMessage(response);
         } else {
-            reloadContent ();

```

Comment:
why are you removing all this?

Reply:
I didn't remove it. If you look above, I moved the call up